src/main/services/ExportService.ts

```
@@ -0,0 +1,894 @@
+  /**
+   * Format date according to options
+   */
+  private formatDate(dateString: string, format: string = 'iso'): string {
+    const date = new Date(dateString);
+    
+    switch (format) {
+      case 'local':
+        return date.toLocaleString();
+      case 'short':
+        return date.toLocaleDateString();
+      case 'iso':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)